picki/app/components/budgetslider/budgetSlider.js

```diff
@@ -0,0 +1,62 @@
+'use client';
+
+import { Range } from 'react-range';
+import '../budgetSlider/budgetSlider.css';
+
+const STEP = 10;
+const MIN = 0;
+const MAX = 500;
+
+export default function BudgetSlider({ values, setValues }) {
+  return (
+    <div className="budget-slider-container">
```

Comment:
if this component is BudgetSlider, styles should be .budget-slider and .budget-slider__container, etc.